src/Components/CardboardListCallout/CardboardListCallout.tsx

```diff
@@ -127,7 +127,16 @@ const CardboardListCallout = <T extends unknown>({
                     <PrimaryButton
                         data-testid={dataButtonTestId}
                         styles={cardboardListCalloutPrimaryButtonStyles}
-                        onClick={primaryActionProps.onPrimaryActionClick}
+                        onClick={(_event) => {
+                            // when where is no search results and if expose search term with primary action is set,
+                            // dispatch the search text within onPrimaryClick action to be consumed for a purpose by the handler
+                            primaryActionProps.isSearchTermExposedWithPrimaryActionClick &&
+                            filteredListItems.length === 0
+                                ? primaryActionProps.onPrimaryActionClick(
+                                      searchText
```

Comment:
can we not simply always pass the search text? Why only do this when the results are 0? If they are clicking create after searching, wouldn't the same assumptions hold that they did not find what they were looking for?